lib/libs/email/content/withdrawPackage/emailTemplates/ChipSpaCMS.tsx

```diff
@@ -22,6 +23,7 @@ export const ChipSpaCMSEmail = ({
         Summary: variables.additionalInformation,
       }}
     />
-    <SpamNotice />
+    <Divider />
```

Comment:
I think since you added the divider to the SpamWarning you should remove it here.

Reply:
Resolving with https://github.com/Enterprise-CMCS/macpro-mako/pull/876/commits/e5f62430b8274f074e9e8107ebd118bada92ab5c